index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require_once 'dbConnect.php';
+require_once 'functions.php';
+
+$db = connectToDb($dbName, $userName, $password);
+
+//set query variable and pass to query function to fetch data from DB
+$sqlQuery = 'SELECT `id`, `supplier_name`, `date`, `amount`,`ccy`, `details`
```

Comment:
move this into inside your newly created getReceipts() function so you dont have to pass it in